tests/files.rs

```diff
@@ -26,3 +26,15 @@ fn test_j219() {
     let bms = Bms::from_token_stream(&ts, RngMock([1])).expect("must be parsed");
     eprintln!("{:?}", bms);
 }
+
+#[test]
+fn test_blank() {
+    let source = include_str!("dive_withblank.bme");
+    let ts = parse(source);
+    assert_eq!(ts.is_err(), true);
```

Comment:
Hmm, I'd better to check whether the actual `Err` is an expected one. And you should remove these commented out codes.

Reply:
That means the actual error type would be implemented `Eq` trait, I'm not sure if you want to do that, but I add it in the code.